ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -53,10 +53,8 @@ public KafkaTopicClientImpl(final AdminClient adminClient) {
     init();
   }
 
-  @Override
-  public void createTopic(final String topic, final int numPartitions, final short
-      replicatonFactor) {
-    log.info("Creating topic '{}'", topic);
+
+  public void createTopic(final String topic, final int numPartitions, final short replicatonFactor) {
```

Comment:
This method overrides a the same method from the interface. It needs ` @Override` annotation.

Reply:
Ah. That is a rebase mistake. Will clean it up.